.env

```diff
@@ -1,6 +1,6 @@
-KILT_IMG=kiltprotocol/peregrine:develop
-KILT_RAW_SPEC_FILE=/node/dev-specs/kilt-parachain/peregrine-kilt-fast-gov.json
-KILT_RUNTIME=mashnet
+KILT_IMG=kiltprotocol/peregrine:0.2.0-69f254e
+KILT_RAW_SPEC_FILE=spiritnet-dev
+KILT_RUNTIME=spiritnet
 
 POLKADOT_RAW_SPEC_FILE=polkadot.raw.json
-POLKADOT_IMG=parity/polkadot:v0.9.5
+POLKADOT_IMG=parity/polkadot:v0.9.7
```

Comment:
new relay chain? is that what we use on wilt?

Reply:
shouldn't affect rococo but that's the latest wilt